Package.swift

```diff
@@ -0,0 +1,223 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import PackageDescription
+
+let package = Package(
+  name: "GoogleUtilities",
+  platforms: [.iOS(.v10), .macOS(.v10_12), .tvOS(.v10), .watchOS(.v6)],
+  products: [
+    .library(
+      name: "AppDelegateSwizzler",
+      targets: ["GoogleUtilities_AppDelegateSwizzler"]
+    ),
+    .library(
+      name: "UserDefaults",
+      targets: ["GoogleUtilities_UserDefaults"]
+    ),
```

Comment:
I wonder if we will eventually need to define a product for each target to make them available to Firebase?